src/pytest/test.py

```diff
@@ -1,33 +1,31 @@
 # -*- coding: utf-8 -*-
 
-from rmtest import BaseModuleTestCase
+from base_case import BaseSearchTestCase
 import redis
 import unittest
 from hotels import hotels
 import random
 import time
 
 
-class SearchTestCase(BaseModuleTestCase):
-    def setUp(self):
-        super(SearchTestCase, self).setUp()
-        self.client.flushdb()
-
+class SearchTestCase(BaseSearchTestCase):
     def testAdd(self):
-        with self.redis() as r:
-            r.flushdb()
-            self.assertOk(r.execute_command(
-                'ft.create', 'idx', 'schema', 'title', 'text', 'body', 'text'))
-            self.assertTrue(r.exists('idx:idx'))
-            self.assertOk(r.execute_command('ft.add', 'idx', 'doc1', 1.0, 'fields',
-                                            'title', 'hello world',
-                                            'body', 'lorem ist ipsum'))
-
-            for _ in r.retry_with_rdb_reload():
-                prefix = 'ft'
-                self.assertExists(r, prefix + ':idx/hello')
-                self.assertExists(r, prefix + ':idx/world')
-                self.assertExists(r, prefix + ':idx/lorem')
+        if self.is_cluster():
+            raise unittest.SkipTest()
+
+        r = self
```

Comment:
why not just renaming 'r' with 'self' ?

Reply:
Smaller delta. Will do at a different time.